src/pages/install/index.md

```diff
@@ -4,25 +4,25 @@ import TabItem from '@theme/TabItem';
 
 # Installing RGBDS
 
-Most people will want to use a release (the latest the bestest!), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
+Most people will want to use a release (the more recent the better), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
 
 The two main options to install a release of RGBDS are to download pre-built executables, or to [build from source](/install/source).
 Below is a list of <abbr title="Operating System">OS</abbr>es for which we know pre-built executables are available.
 
 If your OS isn’t listed below, try finding `rgbds` in your package manager—refer to your OS' documentation for help.
 If you still can’t find RGBDS, or the specific version you are looking for is unavailable, you will have to [compile it from source](#building-from-source).
 
+
 <Tabs>
-<TabItem value="windows" label="Windows">
+<TabItem value="linux" label="Linux">
 
-The install instructions change a bit depending on the environment you wish to use RGBDS with.
+TO DO
 
-:::tip
+</TabItem>
 
-The "plain Windows" instructions may also work for some environments listed further below, since most of them pick up Windows' `PATH`.
-However, if you have such an environment, we recommend you follow its specific install instructions instead.
+<TabItem value="windows" label="Windows">
 
-:::
+The install instructions change a bit depending on the environment you wish to use RGBDS with.
 
 <details>
 <summary>"Plain" Windows (CMD, PowerShell)</summary>
```

Comment:
Actually, the WSL instructions should instruct to refer to the Linux instructions for the corresponding Linux distribution (noting that if you don't know, it's probably Ubuntu).

I'm thinking we should organise the tabs something like `WSL`, then `MSYS2 or Cygwin`, and finally `none of these`. That should make what the last one is about, clearer. I think.

Reply:
This seems a good idea, although I have no experience whatsoever on Windows so if you have any specific ideas on how to reorganize the Windows tab feel free to push

Reply:
I'm not a Windows user anymore, but I think I have some ideas. I'll do that then.

Reply:
OK, done!